docs/reference/templates.rst

```diff
@@ -202,19 +191,17 @@ can specify the templates to use in the ``Admin`` service definition:
 
 Changes made using the ``setTemplate()`` and ``setTemplates()`` methods
 override the customizations made in the configuration file, so you can specify
-a global custom template and then override that customization on a specific
+a global custom template and then override that customization for a specific
```

Comment:
Separate commit here too

Reply:
I agree about the others are grammar fixes, but this is a change about the meaning between the previous and the current behavior.
The previous setting was required **on** a specific admin definition, while the current one is required **for** a specific admin definition.

Reply:
Oh yes, you are definitely right, I overlooked that one 👍